src/plugins/data/public/ui/search_bar/search_bar.scss

```diff
@@ -0,0 +1,4 @@
+.globalQueryEditor,
+.globalQueryBar {
+    display: contents;
+}
```

Comment:
this is very dangerous, its going to make the styling applied to these elements void. That means that every styling we have for these two elements are essentially unapplied, which leads to unmaintainable code.

But i just looked through the codebase and see this is a frustrating problem with no simple fix. I think we should scope this styling to be specific to `explore`. So define & scope this within a explore page styling, and a comment would be good explaining why its so. And I think in the context of explore, we only care about `.globalQueryEditor`.